client/my-sites/marketing/do-it-for-me/difm-lite-in-progress.tsx

```diff
@@ -63,18 +63,18 @@ function WebsiteContentSubmissionPending( { primaryDomain, siteId, siteSlug }: P
 				/>
 			),
 		},
-		args: {
-			contentSubmissionDueDate: contentSubmissionDueDate
-				? moment( contentSubmissionDueDate ).format( 'MMMM Do, YYYY' )
-				: null,
-		},
 	};
 
 	const lineText = contentSubmissionDueDate
 		? translate(
 				'Click the button below to provide the content we need to build your site by %(contentSubmissionDueDate)s.{{br}}{{/br}}' +
 					'{{SupportLink}}Contact support{{/SupportLink}} if you have any questions.',
-				lineTextTranslateOptions
+				{
+					...lineTextTranslateOptions,
```

Comment:
👍 Nice improvement.

FWIW this will no longer choke our custom ESLint rules since #78305 landed.